server/src/tram_track_graph_transformer/tram_track_graph_transformer.py

```diff
@@ -61,6 +67,11 @@ def _build_tram_track_graph_from_osm_ways(self):
 
         return graph
 
+    def _get_node_type(self, node: Node):
```

Comment:
Add a short docstring here:

```suggestion
    def _get_node_type(self, node: Node):
        """
        OSM node IDs mentioned in `custom_stop_mapping` field of `CityConfiguration`
        should always have a type of `NodeType.TRAM_STOP`. This is due to these nodes
        being used for tram stop mapping between GTFS and OSM, which means that
        each node can potentially act as a tram stop, even when it has a different type
        assigned to it by OSM.
        """
```